src/platform/packages/shared/kbn-es/src/serverless_resources/project_roles/chat/roles.yml

```diff
@@ -0,0 +1,55 @@
+# -----
+# Source: project-controller/blob/main/internal/project/esproject/config/roles.yml
```

Comment:
This one is no longer correct (and also points to a private repository 🙈). These are in ES controller now (`../internal/config/roles`)

Reply:
Good point. Updated the comments of all these role definition files with a vague pointer, see 551ceea. I think we still want a way for us to know where this is coming from without actually pointing to an URL. Do you think that works?

Reply:
>Copy from elasticsearch-controller helm values

These are no longer Helm values, maybe something like `Copy from internal roles config in elasticsearch-controller`? People will find them if they need them.

Reply:
Alright, updated in 52b5731 to `Copy from internal roles config in elasticsearch-controller` as you suggested.

Reply:
Looks good, thanks!